CHANGELOG.md

```diff
@@ -210,6 +210,7 @@
 
 ### BREAKING CHANGES
 
+* Introduced [`EMIT CHANGES`](https://docs.ksqldb.io/en/latest/operate-and-deploy/ksql-vs-ksqldb/#syntax) syntax to signify streaming queries. Persistent streaming queries do not yet require an `EMIT CHANGES` clause, but transient streaming queries do.
```

Comment:
Humm.... terminology on different queries may not be inline with our proposed standard.

* 'streaming query' should be 'push query', right?
* 'transient streaming queries' would be... `push query` again, I think.
* 'Persistent streaming queries'... not sure what that is now...

Maybe change:

> Introduced [`EMIT CHANGES`](https://docs.ksqldb.io/en/latest/operate-and-deploy/ksql-vs-ksqldb/#syntax) syntax to signify streaming queries

To:

> Introduced [`EMIT CHANGES`](https://docs.ksqldb.io/en/latest/operate-and-deploy/ksql-vs-ksqldb/#syntax) syntax to differentiate push queries from new pull queries.